### DISCUSSION TITLE
ArcticTracker-ESP32 firmware for T-TWR 2.0 Plus

### DISCUSSION THREAD

**Discussion by lyusupov** [nakhonthai/ESP32APRS_T-TWR#32] [General]
Features: &nbsp; APRS Tracker , Digipeater and igate
Website: &nbsp; https://github.com/Hamlabs/ArcticTracker-ESP32
Releases: &nbsp; https://github.com/Hamlabs/ArcticTracker-ESP32/releases

<br>
<br>

![](https://github.com/Hamlabs/ArcticTracker-ESP32/blob/master/t_twr.jpg?raw=true)

<br>
<br>

<img src="http://hamlabs.no/wp-content/uploads/sites/4/2023/01/Screenshot_20230110_223416.png" width="800">

**Comment by CcRup** [comment]
Is there a quick flasher available for this?
